book/src/exec.md

```diff
@@ -0,0 +1,55 @@
+## On a high level
+Users use intents when they want to exchange their assets for some other asset. High level description of the flow:
+1. **Create**: User creates an intent publishing the information of what they have and what they want to get in exchange
+2. **Gossip**: The intent goes to the intent gossip network and gossiped around the solver nodes.
+3. **Solve**: Solvers find matching intents and update the intent until it is fully satisfied
+4. **Finalize**: When the intent is satisfied, a transaction is created and published on the blockchain
+
+![img_1.png](img/img_exec_high.png)
+
+### Create an intent
+User intents can be seen as partial transactions. Users send them to the intent gossip network and solvers match them in order to create full transactions and publish them on the blockchain. That implies that users need to give some information to the solvers that is sufficient to create the final transaction (proofs). To make sure that users reveal the minimal amount of private infortmation that is still enough to create the transaction, users create **intermediate notes** from the notes they are willing to spend.
+
+#### Intermediate notes
+An intermediate note is a note with a simplified sending VP. This VP is specific to the intent and contains only the rules that are necessary to satisfy for the transaction to happen. All of the note fields stay the same except:
+- sending VP -> intept-specific VP
+- owner address -> intent-specific owner address (sending VP is used to calculate the owner address, so changing the VP implies the change of the owner address)
```

Comment:
Why does the intent-specific VP change?

Reply:
By 'change' here I mean that going from userVP to intent-specific VP will lead to a different owner address in the new note. I'll rephrase that